benchmarks/ImageProcessing/BuddyResize2DBenchmark.cpp

```diff
@@ -0,0 +1,220 @@
+//===- BuddyResize2DBenchmark.cpp -------------------------------------------===//
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//===----------------------------------------------------------------------===//
+//
+// This file implements the benchmark for Resize2D operation.
+//
+//===----------------------------------------------------------------------===//
+
+// #include "Kernels.h"
```

Comment:
Please remove this if you are not using it.

Reply:
Corrected.